specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/cluster.json

```diff
@@ -1009,6 +1058,14 @@
         "msiResourceId": {
           "type": "string",
           "description": "The managed identity (MSI) that is allowed to access the storage account, only to be specified for Azure Data Lake Storage Gen 2."
+        },
+        "saskey": {
+          "type": "string",
+          "description": "The shared access signature key."
+        },
+        "fileshare": {
```

Comment:
The same with saskey, per my understanding,  it should be `fileShare`. Now I want to keep consistent with backend, I will create a work item to track this, and will fix this issue in our new API version.